src/Microsoft.AspNetCore.Server.IISIntegration/Server/IISHttpServer.cs

```diff
@@ -122,11 +144,12 @@ private static NativeMethods.REQUEST_NOTIFICATION_STATUS HandleRequest(IntPtr pI
             var server = (IISHttpServer)GCHandle.FromIntPtr(pvRequestContext).Target;
             Interlocked.Increment(ref server._outstandingRequests);
 
-            var context = server._iisContextFactory.CreateHttpContext(pInProcessHandler);
-
-            var task = Task.Run(() => context.ProcessRequestAsync());
-
-            task.ContinueWith((t, state) => CompleteRequest((IISHttpContext)state, t), context);
+            _ = Task.Run(
```

Comment:
@davidfowl did we want to put this back on the IIS Request thread?

Reply:
No closures...

Reply:
It looks better and taking ctor off iis thread gains ~4k. I say lets take it and iterate more later.

Reply:
Get rid of the closure and use QUWI?

Reply:
There is a task created for state machine anyway

Reply:
My bad, you are right. Will change.

Reply:
But you can avoid the per request allocation....

Reply:
Also, cross compile for nca2.1 and compile out inproc from netstandard